IPython/html/static/edit/js/main.js

```diff
@@ -75,4 +77,14 @@ require([
         }
     };
 
+    // Make sure the codemirror editor is sized appropriatley.
+    var _handle_resize = function() {
+        var header = $('#header');
+        var padding = header.outerHeight(true) - header.innerHeight();
+        $('div.CodeMirror').height(window.innerHeight - header.height() - 2*padding);
```

Comment:
So this uses the padding from the header as the padding for the Codemirror widget? That's probably worth an explanatory comment if so, because it's not obvious why that would happen.